junos/resource_static_route_test.go

```diff
@@ -147,10 +187,23 @@ resource junos_static_route testacc_staticRoute_instance {
   qualified_next_hop {
     next_hop   = "st0.0"
     preference = 101
-    metric     = 101
+	metric     = 101
   }
   community = ["no-advertise"]
 }
+resource junos_static_route testacc_staticRoute_interface {
+  destination      = "192.0.3.0/24"
+  routing_instance = junos_routing_instance.testacc_staticRoute.name
+  preference       = 102
+  metric           = 102
+  qualified_next_hop {
+	next_hop   = "192.0.1.1"
+	preference = 102
+	metric     = 102
+	interface  = "st0.0"
+}
+community = ["no-advertise"]
+}
```

Comment:
Unnecessary new resource. You can just add a second `qualified_next_hop` to the existing resource with only arguments to test : 
```
 qualified_next_hop {
  next_hop   = "192.0.2.250"
  interface  = "st0.0"
}
```
A second resource will extend the test time.

Reply:
Fixed